modules/administration-guide/partials/proc_verification-custom-registries.adoc

```diff
@@ -0,0 +1,73 @@
+// deploying-the-registries
+
+[id="verification-custom_registries-_{context}"]
+
+.Verification steps
+
+. To verify that the new plug-in is correctly published to the plug-in registry, make a request to the registry path `/v3/plugins/index.json` (or `/devfiles/index.json` for the devfile registry).
++
+[subs="+quotes,+attributes"]
+----
+$ URL=$({orch-cli}  get -o 'custom-columns=URL:.spec.rules[0].host' \
+  -l app=che-plugin-registry {platform-routing} --no-headers)
+$ INDEX_JSON=$(curl -sSL http://$\{URL}/v3/plugins/index.json)
+$ echo $\{INDEX_JSON} | grep -A 4 -B 5 "\"name\":\"my-plug-in\""
+,\{
+ "id": "my-org/my-plug-in/1.0.0",
+ "displayName":"This is my first plug-in for {prod-short}",
+ "version":"1.0.0",
+ "type":"VS Code extension",
+ "name":"my-plug-in",
+ "description":"This plugins shows that we are able to add plugins to the registry...",
+ "publisher":"my-org",
+ "links": \{"self":"/v3/plugins/my-org/my-plug-in/1.0.0" }
+}
+--
+--
+,\{
+ "id": "my-org/my-plug-in/latest",
+ "displayName":"This is my first plug-in for {prod-short}",
+ "version":"latest",
+ "type":"VS Code extension",
+ "name":"my-plug-in",
+ "description":"This plugins shows that we are able to add plugins to the registry...",
+ "publisher":"my-org",
+ "links": \{"self":"/v3/plugins/my-org/my-plug-in/latest" }
+}
+----
+
+. Verify that the {prod-short} server points to the URL of the registry. To do this, compare the value of the `pass:[CHE_WORKSPACE_PLUGIN__REGISTRY__URL]` parameter in the `che` ConfigMap (or `pass:[CHE_WORKSPACE_DEVFILE__REGISTRY__URL]` for the devfile registry):
++
+[subs="+quotes,+attributes"]
```

Comment:
```suggestion
[subs="+attributes"]
```

or otherwise CHE_WORKSPACE_DEVFILE__REGISTRY__URL is rendered CHE_WORKSPACE_DEVFILE*REGISTRY*URL